src/host/ToneGenerator.cpp

```diff
@@ -7,12 +7,9 @@ namespace host {
     static double angle = 0.0;
 
     for(int i = 0; i < length; i++) {
-      *stream++ = 255 * std::cos(angle);
-      angle += 3.14159 / 100;
-
-      if(angle > 2.0 * 3.14159) {
-        angle -= 2.0 * 3.14159;
-      }
+      int8_t sample = std::sin(angle) >= 0 ? 1 : -1;
+      *stream++ = 5 * sample;
+      angle += 2 * M_PI / 800;
```

Comment:
Not sure how `M_PI` is defined since we never define `_USE_MATH_DEFINES` anywhere -- could you add this before the `#include <cmath>` line in this file?

```
#define _USE_MATH_DEFINES
#include <cmath>
```